CHANGELOG.md

```diff
@@ -9,7 +9,7 @@ See the [change log guidelines](http://keepachangelog.com/) for information on h
 ### Added
 - Add an option to overwrite content in Geopackage Writer instead of appending content
 - Add a parameter to the GeoPackage writer that allows creating tables for all mapping-relevant target types
-- Add the creation of a .cpg file when a .shp file is beeing exported
+- Create a code page (.cpg) file when exporting a Shapefile
```

Comment:
This change is unrelated to the fixes for the Windows Installer and shouldn't be in the same commit.

Reply:
@florianesser I know...do I need to create a pull request just for a file? Or maybe this should go to the release branch?

Reply:
> do I need to create a pull request just for a file? 

Not necessarily, but it should at least be in a separate commit.

> Or maybe this should go to the release branch?

The change should go into `master`. As the 5.0.1 release is already tagged and built, I would be in favour of not re-opening it for this editorial change.

Reply:
@emanuelaepure10 The commit message `refactor! feat: Release hale»studio 5.0.1` is not consistent with the developer guidelines. For a fixup commit you would use `fixup! feat: Release hale»studio 5.0.1`.